extensions/wikia/Track/Track.php

```diff
@@ -99,19 +99,13 @@ private static function getGAURL( $type, $category, $action, $label, $value, $ti
 	}
 
 	private static function getGATrackingIds() {
-		global $wgDevelEnvironment, $wgStagingEnvironment, $wgIsGASpecialWiki, $wgUser;
+		global $wgDevelEnvironment, $wgStagingEnvironment;
 
 		$tids = [ ];
 		// 10% sampling for general account
 		if ( mt_rand( 0, 9 ) === 0 ) {
```

Comment:
hmm, should we remove this? 
does that mean we don't track at 100% on general account?

Reply:
@hakubo this is tracking on backend side, on frontend it is 100%, I'll change it to 100%